yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -454,10 +478,16 @@ private[spark] class Client(
      *   (3) whether to add these resources to the classpath
      */
     val cachedSecondaryJarLinks = ListBuffer.empty[String]
+    val files = sparkConf.get(FILES_TO_DISTRIBUTE).map(p => Utils.resolveURIs(p))
+      .orElse(sys.env.get("SPARK_YARN_DIST_FILES"))
```

Comment:
No as I pointed out the other jira SPARK-3374 was proposing to remove.  There might be some internal env variables used just to communicate between Client and AM but anything external can be removed for 2.x